build-system/compile/bundles.config.extensions.json

```diff
@@ -352,10 +352,21 @@
   },
   {
     "name": "amp-date-picker",
-    "version": "0.1",
+    "version": "1.0",
     "latestVersion": "0.1",
     "options": {
-      "hasCss": true
+      "hasCss": true,
+      "bento": true
+    }
+  },
+  {
+    "name": "amp-date-picker",
+    "version": "1.0",
+    "latestVersion": "0.1",
+    "options": {
+      "hasCss": true,
+      "bento": true,
+      "npm": true
     }
   },
```

Comment:
This isn't supposed to be duplicated, right?

Reply:
Yep, fixed!